subiquity/controllers/refresh.py

```diff
@@ -58,15 +60,47 @@ def __init__(self, app):
         self.new_snap_version = ""
 
         self.offered_first_time = False
+        self.active = self.interactive()
+
+    def load_autoinstall_data(self, data):
+        if data is not None and data.get('refresh'):
```

Comment:
data.get('refresh-installer')  or 'refresh' ?

Reply:
'refresh'. The format of the yaml is:

refresh-installer:
   refresh: yes